src/components/Tutorial/NavigationBar/SecondaryDropdown.vue

```diff
@@ -11,7 +11,7 @@
 <template>
   <DropdownCustom
     :value="currentOption"
-    aria-label="Current section"
+    :aria-label="$t('tutorials.nav.current', { thing: $t('sections.title') })"
```

Comment:
this includes section #?

Reply:
Section number? It seems it didn't before so I didn't include it in the translation either, but let me know if I'm missing something 🤔

Reply:
Sorry I should've clarified better.
From the locale file:  
```
 "sections": {
    "title": "Section {number}",
    ...
}
```

Does this mean instead of "Current section", VO will instead say "Current Section (number)"? Maybe I misunderstood.